spec/crypt_keeper/model_spec.rb

```diff
@@ -25,8 +25,7 @@
       end
 
       it "allows binary as a valid type" do
-        subject.crypt_keeper :storage, encryptor: :fake_encryptor
-        allow(subject.columns_hash['storage']).to receive(:type).and_return(:binary)
+        subject.crypt_keeper :storage_binary, encryptor: :fake_encryptor
```

Comment:
The original test added coverage to make sure binary data was being saved/retrieved. The new test is only asserting that the value is truthy. Can this test be updated to make sure the storage piece is working as expected?

Reply:
@jmazzi yep, can do.

Reply:
@jmazzi done! I considered wrapping the whole `context "Encryption and Decryption"` in a loop to iterate through each field type, but I felt that even through it was more DRY it reduced clarity so I settled on this. Let me know if you'd prefer the former.